### ISSUE TITLE
Add a "/" or "s" icon in search bar

### ISSUE THREAD

**Issue by thatlittleboy** [squidfunk/mkdocs-material#4926]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I want to suggest an idea and checked that ...

- [X] ... to my best knowledge, my idea wouldn't break something for other users
- [X] ... the documentation does not mention anything about my idea
- [X] ... there are no open or closed issues that are related to my idea

### Description

The main idea is to add a small icon containing the keyboard shortcut to trigger the site-search to the search bar, something like what **Github** or **Docusaurus** does. Could be "/" or "s", either is fine.  

![image](https://user-images.githubusercontent.com/30731072/215236488-07df3e1f-e8e7-4013-8e15-e2b99ed3d727.png)

![image](https://user-images.githubusercontent.com/30731072/215237088-4759a607-be29-43cf-ad60-bfe49a796235.png)

I think it's also a better use of the screen real estate since, the search bar is a text box and already taking up some horizontal space in the top bar (as opposed to say, some sphinx docs/themes that only have a search icon, not a search bar, in the initial state). We might as well use some of that space to inform users of a way to trigger search without leaving their keyboard.  

**Details**:
- If the search bar comes into focus, the svg icon should disappear to make room for text to be displayed in full.
- Can consider making the display of this icon configurable in `mkdocs.yml`, if necessary.

Related links:
- https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-search/#built-in-search-plugin

### Use Cases

It gives our users a visual cue that they can use a [keyboard shortcut (/ or s)](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#keyboard-shortcuts) to trigger a search instead of using their mouse to click the search bar.

In short, it improves discoverability of a very useful keyboard shortcut.

### Screenshots / Mockups

![image](https://user-images.githubusercontent.com/30731072/215236988-54a38a61-63f1-4f03-9b27-92d9c6c0eeb1.png)

Very ugly mockup, but you get the idea :)

**Comment by squidfunk** [comment]
Thanks for suggesting! Duplicate of #2574 with a solution based on customization in https://github.com/squidfunk/mkdocs-material/issues/2574#issuecomment-821979698 😊
